include/mbedtls/psa_util.h

```diff
@@ -91,6 +91,7 @@ static inline psa_algorithm_t mbedtls_psa_translate_cipher_mode(
             if( taglen == 0 )
                 return( PSA_ALG_CBC_NO_PADDING );
             /* Intentional fallthrough for taglen != 0 */
+            /* fallthrough */
```

Comment:
```
$ gcc --version
gcc (Ubuntu 7.3.0-27ubuntu1~18.04) 7.3.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Reply:
Oh. Wait, I'm still confused: reading the [documentation](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-fallthrough_003d), I don't see a setting that recognizes `/* fallthrough */` but not `/* foo fallthrough bar */` — it should be `/*-fallthrough*/` or `/*@fallthrough@*/`.

Reply:
Maybe it's because of Fallthrough vs fallthrough (case-sensitive)

Reply:
Ah, yes, got it. The last bullet point for `-Wimplicit-fallthrough=3` allows `fallthrough` with optional surrounding whitespace, together with a few other variants. (And the previous two bullet points allow all-uppercase or titlecase variants.)

Reply:
@gilles-peskine-arm Or we could even go with
```
/* fallthrough */ /* for compilers */
```
(I'm not even kidding. (I think.))

@AndrzejKurek We'll need this side-ported to TLS. Perhaps this could be part of 2395? Either way, please let @k-stachowiak know as he's been assigned to fix this on the TLS side.

Reply:
Done.